nmdc_server/api.py

```diff
@@ -746,6 +751,33 @@ async def update_submission(
     crud.update_submission_lock(db, submission.id)
     return submission
 
+def create_github_issue(submission,user):
+
+    url = "https://api.github.com/repos/JamesTessmer/Issue-bot-testing/issues?state=all"
+
+    cookies = {'logged_in':'no'}
+
+    headers = {'Authorization':'Bearer TOKEN',
+            'Content-Type': 'text/plain; charset=utf-8'}
+    print(submission.metadata_submission)
+    studyform = submission.metadata_submission['studyForm']
+    contextform = submission.metadata_submission['contextForm']
+    multiomicsform = submission.metadata_submission['multiOmicsForm']
+    pi = studyform['piName']
+    piorcid = studyform['piOrcid']
+    datagenerated = contextform['dataGenerated']
+    omicsprocessingtypes = multiomicsform['omicsProcessingTypes']
+    sampletype = submission.metadata_submission['templates']
+    sampledata = submission.metadata_submission['sampleData']
+    numsamples = 0
+    for key in sampledata:
+        numsamples = numsamples + len(sampledata[key])
+
+    payload = '{\n "title":'+f'"NMDC Submission: {submission.id}", \n "body":"Submitter: {user.orcid} \\n Submission ID: {submission.id} \\n Has data been generated: {datagenerated} \\n PI: {pi} {piorcid} \\n Status: Submitted -Pending Review \\n Data types: {omicsprocessingtypes} \\n Sample type: {sampletype} \\n Number of samples: {numsamples} \\n Note:", \n "assignees": ["JamesTessmer"], \n "labels":["testing"]'+'}'
```

Comment:
Those are good ideas. I was focused on getting everything working, so now I can make it readable. I haven't done much with json in python so I actually didn't know that existed but looking at the docs it does seem like it would fit here really well.

Reply:
I've added this suggestion. Everything looks far more readable now!